template.md

```diff
@@ -0,0 +1,40 @@
+## Title of the enehancement proposal
+
+<!-- Add link to issue/epic if available -->
+
+## Summary
+
+<!-- Add brief summary of enhancement proposal. -->
+
+## Proposal
+
+<!-- Detailed information about the enhancement proposal. Add any subsections if needed. -->
+
+## Design Details
+
+<!-- Detailed design of the proposal. This should include any yamls, CRDs, data structures, APIs etc. Add/remove any subsections
+if needed -->
+
+### Pros
+<!-- Optional -->
+
+### Cons
+<!-- Optional -->
+
+### Backward Compatibility
+
+<!-- Optional. Any backward compatibility concerns -->
+
+### Alternate Design
+
+<!-- Optional. Any alternate designs considered, if applicable. -->
+
+## Owners
+
+<!-- Optional. List of owners for this enhancement proposal and its implementation -->
+* Owner 1 Name <owner1@email.com>
+* Owner 2 Name <owner2@email.com>
+
+## Work Items
```

Comment:
IMO yes, we can just copy the `work items` from here to the epic issue

Reply:
That is why I mentioned it to be Optional. This will be useful if we have some proposals which don't have a corresponding Epic.

Reply:
I think that work items are not in the scope of the proposal, even if no epic is open yet.

Reply:
> That is why I mentioned it to be Optional. This will be useful if we have some proposals which don't have a corresponding Epic.

Do we intend to support EPs with no epic?

To me it seems each EP is a design proposal for implementing an epic, and wouldn't expect an EP without one.